apps/cugetreg-web/src/pages/server-site.xml/index.tsx

```diff
@@ -51,4 +51,5 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
 }
 
 // Default export to prevent next.js errors
-export default () => <div />
+const Sitemap = () => <div />
+export default Sitemap
```

Comment:
is there a reason for this?

Reply:
Use in Google Search Console to tell Google what pages we have

Reply:
No

Reply:
เอ๊ะ ไม่แน่ใจว่า export default เป็น anonymous ได้ไหม ถ้าไม่ ก็ต้องทำแบบนี้แหละ

Reply:
@saenyakorn ถ้าไม่ทำ eslint จะด่าครับ ฝากกด approve ด้วยครับ 🙏